src/clj/harja/palvelin/palvelut/budjettisuunnittelu.clj

```diff
@@ -381,16 +381,16 @@
                                 (doseq [{:keys [vuosi kuukausi osa-kuukaudesta tunnit tuntipalkka]} uudet-jhkt]
                                   (insert! db
                                            ::bs/johto-ja-hallintokorvaus
-                                           {::bs/urakka-id urakka-id
-                                            ::bs/toimenkuva-id toimenkuva-id
-                                            ::bs/tunnit tunnit
-                                            ::bs/tuntipalkka tuntipalkka
-                                            ::bs/kuukausi kuukausi
-                                            ::bs/vuosi vuosi
-                                            ::bs/ennen-urakkaa ennen-urakkaa?
+                                           {::bs/urakka-id       urakka-id
+                                            ::bs/toimenkuva-id   toimenkuva-id
```

Comment:
Tää toinen ikuinen formatointiasia. Omat asetukset formatoi mäpit tolla alkuperäisellä, ja joskus way back eri henkilöiden kesken se valittiin "the harja way". Nyt voidaan altiistaa asia uudelleen tiimin päätettäväksi, olisi kivaa että me kaikki tuotettais samanlaista formaattia luonnostaan niin vähenee noise kommiteissa